checkstyle.xml

```diff
@@ -228,7 +228,7 @@
             <property name="allowThrowsTagsForSubclasses" value="true"/>
         </module>
         <module name="MethodName">
-            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9_]*$"/>
+            <property name="format" value="^[a-z][a-zA-Z0-9][a-zA-Z0-9_]*$"/>
```

Comment:
I think this was right as it was. Method names shouldn't start with a capital letter.

Reply:
That still applies. This just makes it so that method names only have to begin with a single lower-case letter instead of two.

Reply:
Okay.